plugins/openai_plugin.py

```diff
@@ -50,15 +52,23 @@ def request_http(self, query: dict, user_id: int, test_end_time: float = 0):
 
         headers = {"Content-Type": "application/json"}
 
-        data = {
-            "prompt": query["text"],
-            "max_tokens": query["output_tokens"],
-            "min_tokens": query["output_tokens"],
-            "temperature": 1.0,
-            "top_p": 0.9,
-            "seed": 10,
-        }
-
+        if "/v1/chat/completions" in self.host:
+            data = {
+                "messages": [
```

Comment:
As discussed, it might be better to change how dataset.py works so that the system prompt from the dataset is passed to the plugin so that we can put the system prompt in a message from the system role. This is a slightly wider change that can be left for a follow-up PR.